execution/execution.py

```diff
@@ -532,6 +560,39 @@ def swatch_list_all():
 
     return dic_full_list
 
+def is_fresh(stale_timestamp):
```

Comment:
@starnold-redhat 
Here we can see in the code as well.

Once you keep only the ansible stuff, I'll review and merge it.

Thank you
Waldirio